cheri_compressed_cap_common.h

```diff
@@ -111,11 +111,11 @@ _CC_STATIC_ASSERT(_CC_N(MAX_RESERVED_OTYPE) <= _CC_N(MAX_REPRESENTABLE_OTYPE), "
 
 // Forward-declare the accessors since we use them inside the struct body:
 struct _cc_N(cap);
-static inline uint8_t _cc_N(get_flags)(const struct _cc_N(cap)* cap);
-static inline uint32_t _cc_N(get_otype)(const struct _cc_N(cap)* cap);
-static inline uint32_t _cc_N(get_perms)(const struct _cc_N(cap)* cap);
-static inline uint8_t _cc_N(get_reserved)(const struct _cc_N(cap)* cap);
-static inline uint32_t _cc_N(get_uperms)(const struct _cc_N(cap)* cap);
+static inline uint8_t _cc_N(get_flags)(const struct _cc_N(cap) * cap);
```

Comment:
clang-format seems confused by these?

Reply:
Yeah I know - I figured I'd just deal with the broken formatting rather than adding `//clang-format off`. Can do that if you prefer?

Reply:
(Perhaps we should hoist the typedef and #define?)

Reply:
Yes that's what I've done now.